atd-vzv/src/App.css

```diff
@@ -2,6 +2,9 @@
   display: flex;
   width: 100%;
   align-items: stretch;
+  min-height: 100vh;
+  /* Mobile viewport iOS Safari bug fix for Content component (Map) */
```

Comment:
would it make sense to add in the commented code a link to that article you sent us as a reference?

Reply:
Definitely - added this for some context about the fix. 👍